script.js

```diff
@@ -180,3 +180,15 @@ cardButton.forEach((cardButtonItem, index) => {
     });
   });
 });
+
+function formValidation(e) {
+  const email = document.querySelector('#youremail');
+  if (email.value !== email.value.toLowerCase()) {
+    document.querySelector('.form-validation').innerHTML = '* Invalid Email !!! All character on the email should be in lowercase !!*';
+    e.preventDefault();
+    e.stopPropagation();
+  }
+}
+
+const form = document.forms['.form'];
+form.addEventListener('submit', formValidation);
```

Comment:
- [ ] Excellent work 👏, @anita00001. However, the email validation is not working properly; we can currently submit the form even if the email contains an uppercase character. Please fix this so that when someone tries to send an email with uppercase characters, they receive an error message. 👌